docker/.env.sample

```diff
@@ -27,7 +27,7 @@ AUTH0_CLIENT_SECRET="YOUR AUTH0 CLIENT SECRET HERE"
 # Superset user registration config
 USER_ROLE="Gamma"
 # Optional, comma-separated list "(),(),()..." of valid permissions for your user role
```

Comment:
Every time I look at this code, I have to re-calibrate to what is going on.  I think this comment needs to contain more info, for example:

- they are always 2-tuples: a permission name and a view name
- where to find a list of allowed values for permissions (if possible)
- where to find a list of allowed values for view names (if possible)

Also the repetition of (all_database_access,all_database_access) strikes me as weird, but, after some digging in Superset, I am at least somewhat convinced that they really did use the same string for both permission name and view name, so the repetition IS necessary.

Reply:
> Also the repetition of (all_database_access,all_database_access) strikes me as weird, but, after some digging in Superset, I am at least somewhat convinced that they really did use the same string for both permission name and view name, so the repetition IS necessary.

That is correct. I have confirmed this by looking into their codebase, and through repeated testing.